frontend/src/services/pdfExport.ts

```diff
@@ -46,6 +88,35 @@ function replaceSpecificDivsWithTextContent(htmlString: string, selector: string
   }
 }
 
+function getApplicationVersion() {
```

Comment:
Moet dit niet een async functie zijn? En suggestie om ook op te geven wat de functie voor type data terug geeft:
```suggestion
async function getApplicationVersion(): Promise<string> {
```